updated output saving module following suggestions from issue #339

AnniekStok opened the pull request:
I have made updates following Tischi's suggestions here: #339

tischi commented on _modules/output_saving.md:
```diff
@@ -16,26 +16,26 @@ objectives:
-  - Open the blobs image (File > Open Samples > Blobs).
-  - Binarize it and run the 'analyze particles' command to generate different kinds of output.
-  - Save the different output types.
+  - Open the image [blob.gif](http://imagej.net/images/blobs.gif) and binarize it.
```
@AnniekStok Could you please one of the images from our repository. In image_data there should be more than enough images with some nuclei that you could use.

AnniekStok replied:
@tischi, I changed it now. If you agree, can you please merge? Thanks!

tischi reviewed (commented):
Perfect. Thanks a lot!
I found one more thing to fix.

AnniekStok commented:
I updated the figure with the version from #351 for now. I also changed the image data so that the blobs image is not used. @tischi  can you please merge?

AnniekStok commented:
Was there any specific reason why the code layout for macro and Jython has changed everywhere? I liked it more before, this seems less clear. Sorry if I missed it
![image](https://user-images.githubusercontent.com/32863964/158967024-ba3532a7-8c29-46b2-bf64-b6a4f7628cc1.png)

tischi commented:
> Was there any specific reason why the code layout for macro and Jython has changed everywhere? 

Yes, see discussion here: https://github.com/NEUBIAS/training-resources/issues/344

Please post there examples where the current rendering does not look nice. When I tested it, it looked good.

AnniekStok commented:
I see now, I just have a typo in that line so that it is not a proper comment. Fixed that now.
